lib/helper/Puppeteer.js

```diff
@@ -1090,7 +1090,11 @@ class Puppeteer extends Helper {
     for (const modifier of modifiers) {
       await this.page.keyboard.down(modifier);
     }
-    await this.page.keyboard.press(key);
+    try {
+      await this.page.keyboard.press(key);
+    } catch (e) {
+      await this.page.keyboard.type(key);
```

Comment:
I think it incorrect) 
Why not be use fillField?

Reply:
If you are inside an input/textarea, and navigate the cursor to a specific location to insert text there. Then, you cannot use fillField (which would overwrite existing content) or appendField (which would always jump to the end of the input/textarea).

Reply:
yes, I agree pressKey can be more useful in this case